src/renderer/views/multiplayer/custom.vue

```diff
@@ -191,6 +293,23 @@ if (active.value) {
     intervalId.value = window.setInterval(api.session.updateBattleList, 5000);
 }
 
+function onRowSelect(event: DataTableRowDoubleClickEvent) {
+    const data = event.data as ScoredSpadsBattle;
+    const scoreExplanation = `\
+Score explanation for ${data.battleOptions.title}
+Primary Factor: ${data.primaryFactor}
+Total score: ${data.score}
+Factors: ${Object.entries(data.factors).length}
+${Object.entries(data.factors)
+    .map(([factorName, factorScore]) => {
+        return `${factorName.padEnd(20, " ")}: ${factorScore.toFixed(2)}`;
+    })
+    .join("\n")}
+`;
+
+    console.log(scoreExplanation, event.data);
+}
+
```

Comment:
I get that this is for debugging, but I think it should be removed before PR is merged

Reply:
Not sure it's great as a tooltip, or maybe I just formatted it weirdly
![image](https://github.com/beyond-all-reason/bar-lobby/assets/1368558/51732cac-7869-4a78-bb19-238331f938ab)

Reply:
Eh, changed my mind. I was thinking just display the final score, but even that's not really relevant to end users. This function is useful to devs, but maybe just comment out the `console.log` for now and add a note saying what the function has been left in for